packages/block-editor/src/components/block-list/block.native.js

```diff
@@ -111,32 +115,46 @@ class BlockListBlock extends Component {
 			isValid,
 			showTitle,
 			title,
+			displayToolbar,
+			parentId,
 		} = this.props;
 
 		const borderColor = isSelected ? focusedBorderColor : 'transparent';
 
 		const accessibilityLabel = this.getAccessibilityLabel();
 
 		return (
-			<TouchableWithoutFeedback
-				onPress={ this.onFocus }
-				accessible={ ! isSelected }
-				accessibilityRole={ 'button' }
-			>
-				<View style={ [ styles.blockHolder, borderStyle, { borderColor } ] }>
-					{ showTitle && this.renderBlockTitle() }
-					<View
-						accessibilityLabel={ accessibilityLabel }
-						style={ [ ! isSelected && styles.blockContainer, isSelected && styles.blockContainerFocused ] }
-					>
-						{ isValid && this.getBlockForType() }
-						{ ! isValid &&
+			<>
+				{ displayToolbar &&
+					<FloatingToolbar>
+						<ToolbarButton
+							title={ __( 'Navigate Up' ) }
+							onClick={ () => this.props.onSelect( parentId ) }
+							icon="arrow-up-alt"
```

Comment:
I won't be able to find the same icon as on given [design] (https://www.figma.com/proto/SptxS8feaNYtWUzoeNM4qI/Nested-Blocks-i2?node-id=82%3A356&viewport=-1433%2C80%2C1&scaling=min-zoom). Anyone know the name of its icon or it needs to be add?

Reply:
@iamthomasbishop we'll need the hierarchy up icon for this, could you be able to provide that one?